juno/assetlist.json

```diff
@@ -1218,7 +1218,28 @@
       "symbol": "CATMOS",
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/catmos.png"
-      }      
+      },
+      {
+      "description": "Inspired by XRP fast network  , token Utilit) donation, memes, and gambling.",
```

Comment:
this description reads strangely...

Reply:
sorry, I am way out of my head trying to get that sorted ,

"description": "Inspired by XRP Utility for replacing swift , WXRPS can be used to donation - NFT Memes - Lottery tickets "